financial-templates-lib/helpers/GasEstimator.js

```diff
@@ -61,9 +61,8 @@ class GasEstimator {
     try {
       const response = await fetch(url);
       const json = await response.json();
-      // The number returned by EthGasStation is a Gwei amount, scaled by 10.
       if (json.fast) {
-        let price = json.fast / 10;
```

Comment:
This is just a guess looking at the JSON response from `Etherchain`, but the Etherchain values all look 10x larger than the EthGasStation values. Moreover, are they possibly divided by `json.block_time`?

Reply:
EthGasStation scales everything by 10x. etherchain does not need to do this :)

Reply:
Ah I confused the two. Good stuff